packages/python/test/integration-1.test.ts

```diff
@@ -1,15 +1,14 @@
-const fs = require('fs');
-const path = require('path');
+require('./integration-setup')(1);
+
+// NOTE: The fixture `00-requrest-path` has special case handling, which is why
```

Comment:
```suggestion
// NOTE: The fixture `00-request-path` has special case handling, which is why
```

Took me a second to realize why I couldn't find it 😆